samples/bluetooth/channel_sounding_ras_initiator/README.rst

```diff
@@ -0,0 +1,92 @@
+.. _channel_sounding_ras_initiator:
+
+Bluetooth: Channel Sounding Initiator with Ranging Requestor
+############################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to use the ranging service to request ranging data from a server.
+It also provides a basic distance estimation algorithm to show how channel sounding can be used to estimate distance between two devices.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also requires a device running a Channel Sounding Reflector with Ranging Responder to connect to.
```

Comment:
Does any CS Reflector with Ranging Responder work, or does it have to be the other Nordic sample? This line (and the corresponding line in the other sample) is then either:
```suggestion
The sample also requires a device running a Channel Sounding Reflector with Ranging Responder to connect to, such as the :ref:`channel_sounding_ras_reflector` sample.
```
or
```suggestion
The sample also requires a device running the :ref:`channel_sounding_ras_reflector` sample to connect to.
```

Reply:
Went with the first option